main.py

```diff
@@ -131,6 +131,61 @@ def calculate_interest(self, rate: float) -> None:
         print("Debit accounts do not earn interest.")
 
 
+class SavingsAccount:
+    def __init__(self, account_number: str, balance: float = 0) -> None:
```

Comment:
czy balance ma być float - założenie? problem z przecinkiem lepiej użyć int i działać na groszach / centach i przeliczać ew przy wyświetlaniu czy numer konta powinien być str? jeżeli tak to może _id a nie _number (nazwa)